lib/testsuite/index.js

```diff
@@ -1,5 +1,5 @@
 const AssertionError = require('assertion-error');
-const {By, Key, locateWith, withTagName} = require('selenium-webdriver');
+const {By, Key, locateWith, withTagName, Browser} = require('selenium-webdriver');
```

Comment:
I think this is not required

Reply:
fixed in #[affe81e](https://github.com/nightwatchjs/nightwatch/pull/3609/commits/affe81e2780078d7260e32ec5733022ad15f2614)